README.md

```diff
@@ -146,6 +146,10 @@ For a debug build:
 Test
 ----
 
+If a ruby was found during configuration, execute the following command before running tests:
```

Comment:
There are suggestions at http://stackoverflow.com/questions/733475/cmake-ctest-make-test-doesnt-build-tests, but they're kind of clunky.